### DISCUSSION TITLE
Lotmaxx shark V2 - Does anyone already configure LaserGRBL with this laser

### DISCUSSION THREAD

**Discussion by rlasjunies** [arkypita/LaserGRBL#1545] [Q&A]
Hi,
Everythink is in the title.
Just bought a Lotmaxx shark V2 who has the capability to laser engrave.
I do not know the configuration of laserGRBL. Your help is welcome ;-)

**Comment by gmmanonymus111** [comment]
Lotmaxx specifies the following settings for engraving:
![image](https://user-images.githubusercontent.com/9658243/127870721-3f7efeb2-127c-469b-88f5-2864e8d462a1.png)

Since this engraver has an STM32 based motherboard, I'm not really sure it has GRBL firmware (and judging by  their own software, I'm not sure it has any LaserGRBL-compatible firmware) on it.
If you can connect it to LaserGRBL, then you can choose settings from LaserGRBL's material DB. (I'd go with the 1,5W Ortur laser as that's the closest to your 1,6W one)

**Comment by rlasjunies** [comment]
Thank for the information, il will try your setup when I'll back at home, at the end of rhe week.

About the firmware, I found that Lotmaxx use Marlin and it's easy to configure in laserGRBL. Selecting this firmware, the printer can communicate natively, via USB. Found it very convenient.

Now I've to learn how to engrave. I tried a picture without great success. By any chance, do you know a food tutorial to engrave picture?

Thanks again.

R

Richard LASJUNIES

________________________________

**Reply by gmmanonymus111** [reply]
I don't quite get what 'food tutorial' is, but I'd take a look at LaserGRBL's usage manual:
[Raster import](https://lasergrbl.com/usage/raster-image-import/)
[Immport parameters](https://lasergrbl.com/usage/raster-image-import/import-parameters/)
[Dithering](https://lasergrbl.com/usage/raster-image-import/dithering-tool/)
[Target image options](https://lasergrbl.com/usage/raster-image-import/target-image-size-and-laser-options/)
For the best result, follow the manual in order it is linked 😛
`Vectorization`, `centerline` and `line-to-line` have similar settings to `dithering`, when you feel ready to try them, relevant info can be found on [lasergrbl's website](https://lasergrbl.com/) under `usage` -> `raster image import` menu

**Comment by rlasjunies** [comment]
Thanks a lot for your proposals I'll follow them with attention.

Richard

Ps: Sorry for my typo error ... Food tuto, versus Good tuto 😅. May be it was time to go to eat at that time 😏

Richard LASJUNIES

________________________________

**Comment by DaGweeds** [comment]
I have the Shark v2 myself. LaserGBRL works fantastically in all cases, except for line to line tracing. There was a change in the way it handled the GCode back in 2018. The fix that was stated in #812 didn't help.


There are a bunch of items I changed for my use. 

Here are my config files that you can place on your machine for testing:
https://bit.ly/3APONHI

Place both the CustomButtons.bin and LaserGBRL.Settings.bin files in:
c:\users\<username>\AppData\Roaming\LaserGRBL

Here's what I changed so you know what you're getting:

In settings:
1. On the protocol tab, change to Marlin
2. On the Raster import tab, turn on Support PWM
3. On Jog control tab, check Show Z up/down control
4. On the GCode tab, I made changes to the header based on how high the laser is when I focus it
![Screenshot 2021-10-15 153535](https://user-images.githubusercontent.com/92604240/137544010-d7564c48-503e-46e1-8fa9-d42fff5a7344.png)

I then changed the definition of a bunch of the buttons to work with this printer as well.

Please note that in both the GCode Header and the "Autofocus" buttons, I have set the focus height of 15mm as that is where my laser focuses. You will need to determine your height and set it accordingly on those two locations.

**Reply by 1Canna** [reply]
Hey G,
Could I please get a new link, for your shared config?
